.github/workflows/black.yaml

```diff
@@ -8,10 +8,10 @@ jobs:
         runs-on: ubuntu-22.04
         steps:
             - uses: actions/checkout@v2
-            - name: Set up Python 3.10
+            - name: Set up Python 3.11
```

Comment:
I'm not sure about this change.  I think for the `black` and `coverage` actions maybe we should leave them on the older versions, until people transition to the newer ones.

Thoughts?

Reply:
I thought these GH actions, minus the python-version-specific unit tests, always used "the latest" python (since most of us are on 3.9 and this is being run on 3.10). Happy to leave it at 3.10 or switch it to 3.9.